scapy/utils.py

```diff
@@ -33,11 +33,11 @@
 ###########
 
 def get_temp_file(keep=False, autoext=""):
```

Comment:
Could you add the following docstring ?
"""
Create a temporary file and return its name. When keep is True, the filename is stored in conf.temp_files
"""